dev_tools/pr_moniter.sh

```diff
@@ -31,10 +31,19 @@
 # the body of the PR's initial message/comment. Users/admins should edit the title and
 # initial comment to appropriately describe the PR.
 #
+# This script will also automatically label PRs based on their code size. The following
```

Comment:
Renaming this script requires syncing with Fedor w.r.t. how the bot is run.

Reply:
Oh I thought @balopat had set this up on cloud using some of his Kubernetes wizardry. @fvkg What do you think ?

Reply:
Yes, I set this up and it is described in https://github.com/quantumlib/Cirq/tree/master/dev_tools/auto_merge.
Namely, rename the script in the Dockerfile here:  https://github.com/quantumlib/Cirq/blob/10c78c929b3c636fb8878a5d8a8e23d123bdca84/dev_tools/auto_merge/Dockerfile#L37

Note though that the whole folder is named auto_merge, so for consistency you might want to rename the whole thing to `pr_monitor` (not `pr_moniter`), or `cirq_bot`...